frontend/app/components/api/api-v3/hal-resources/work-package-resource.service.ts

```diff
@@ -122,7 +123,6 @@ export class WorkPackageResource extends HalResource {
 
   public $embedded: WorkPackageResourceEmbedded;
   public $links: WorkPackageResourceLinks;
-  public id: string;
```

Comment:
The change is only a byproduct of this PR in the sense that it shouldn't be possible to ever modify the unique attribute that is used for identifying a resource, as it will break the connection between its state.

This particular issue stems from an `angular.copy` being used on the resource itself, which is just wrong.